monkey/tests/unit_tests/infection_monkey/exploit/test_powershell.py

```diff
@@ -15,57 +14,57 @@
 PASSWORD_LIST = ["pass1", "pass2"]
 LM_HASH_LIST = ["bogo_lm_1"]
 NT_HASH_LIST = ["bogo_nt_1", "bogo_nt_2"]
-DROPPER_TARGET_PATH_32 = "C:\\agent32"
 DROPPER_TARGET_PATH_64 = "C:\\agent64"
 
-Config = namedtuple(
-    "Config",
-    [
-        "exploit_user_list",
-        "exploit_password_list",
-        "exploit_lm_hash_list",
-        "exploit_ntlm_hash_list",
-        "dropper_target_path_win_64",
-    ],
-)
-
 
 class AuthenticationErrorForTests(Exception):
     pass
 
 
+@pytest.fixture
+def powershell_arguments():
+    options = {
+        "dropper_target_path_win_64": DROPPER_TARGET_PATH_64,
+        "credentials": {
+            "exploit_user_list": USER_LIST,
+            "exploit_password_list": PASSWORD_LIST,
+            "exploit_lm_hash_list": LM_HASH_LIST,
+            "exploit_ntlm_hash_list": NT_HASH_LIST,
+        },
+    }
+    arguments = {
+        "host": VictimHost("127.0.0.1"),
+        "options": options,
+        "current_depth": 2,
+        "telemetry_messenger": MagicMock(),
+        "agent_repository": MagicMock(),
+    }
+    return arguments
+
+
```

Comment:
Why not just make this a constant?

Reply:
I don't think it makes a really big difference. Maybe I will change it.